index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <title>Portfolio: Setup and Mobile First</title>
+</head>
+<body>
+    <main class="header-background-mobile">
```

Comment:
- [ **OPTIONAL** ] For best tags semantic tags I think it would be better if the `<main>` is the container all the sections and not for the entire code base.
Below is an example of good semantics HTML code(how to organize your sections):
```HTML
<header>
   code for the header...
</header>

<main>
   <section>
     code for section1...
  </section>

  <section>
     code for section2...
  </section>
  ....
</main>

<footer>
   code for the footer...
</footer>
```